widget/entry.go

```diff
@@ -17,8 +18,9 @@ import (
 )
 
 const (
-	multiLineRows = 3
-	wordSeparator = "`~!@#$%^&*()-=+[{]}\\|;:'\",.<>/?"
+	bindIgnoreDelay = time.Millisecond * 100 // ignore incoming DataItem fire after we have called Set
```

Comment:
Would this end up ignoring data updates from user code that happen more frequently than 100 ms?

Reply:
It's not ignoring changes faster than 100ms - it is ignoring external updates for a widget being edited by the user, for 10th sec.
I'm not sure if that is what you meant, but given a typical user typing speed this seemed reasonable.

It is possible that this will fail to register correctly if you modify the entry and then 50ms later a remote source changes the data, for these minute corner cases, and for code cleanliness, I opened the other ticket where we should try and find a solution that is not time-based but will be much further reaching.